### PR TITLE
License and corrections

### PR DESCRIPTION
**Pull request by zimmski**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1 @@
+See the gonum AUTHORS file at https://github.com/go-clang/license/blob/master/AUTHORS.
```
Comment (sbinet) [inline AUTHORS]: s/gonum/go-clang/g :)

File: CONTRIBUTORS [hunk 2]
```diff
@@ -0,0 +1 @@
+See the gonum CONTRIBUTORS file at https://github.com/go-clang/license/blob/master/CONTRIBUTORS.
```
Comment (sbinet) [inline CONTRIBUTORS]: s/gonum/go-clang/g :)

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1 @@
+See the gonum LICENSE file at https://github.com/go-clang/license/blob/master/LICENSE.
```
Comment (sbinet) [inline LICENSE]: s/gonum/go-clang/g :)

**Comment by sbinet** [comment]
LGTM once modifications applied.

File: AUTHORS (hunk 1 above)
Comment (zimmski) [inline AUTHORS]: Yep, just saw that ^^

**Pull request merged**